### DISCUSSION TITLE
“Your Love” feat. Du?

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#479] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/your-love/

**Comment by angellisy** [comment]
Bons tempos em que os desenhos antigos do Cartoon eram bobos na medida certa. Hoje qualquer tranqueira entra na grade

**Comment by brunoaugusto** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [terça-feira, 31 de maio de 2022](<https://www.vish.blog.br/videos/your-love/#comment_1142239598>):

> Bons tempos em que os desenhos antigos do Cartoon eram bobos na medida certa. Hoje qualquer tranqueira entra na grade

Pior é ler artigos dizendo que a grade é montada após "pesquisa de opinião". Até parece...

**Comment by one-eyed-willy** [comment]
É pra ver esse tipo de coisa que eu pego Internet do meu vizinho
